### PR TITLE
Add clk.ink (clk.asia) bypass

### LINKED ISSUE #405: Skincarie / clk.ink (CLICKSFLY)
### Domain

Skincarie

### Link

https://skincarie.com/GTaj7Tv

### Version

0.0.0

### What browsers are you seeing the problem on?

Chrome

### What OS are you seeing the problem on?

Windows

### (Optional) Anything else?

Down the rabbit hole of cancerous ad redirects.
1   https://clk.ink/GTaj7Tv
2   https://clk.ink/hbzk

leads to
https://mega.nz/file/Yzp1XB5J#P7vObys5bPOu9Xt0NcmQ92yjbcqCEsUx8v_7D2JfpbI
https://mega.nz/file/UvhXyYKL#WBbEhx23MzjljR3PdEqRhD0RdaorGToVLt__XwMQsiw

I think one of those might've been submitted, but I'm making an issue anyway just in case.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by unkn4wn**
<details> <summary>NOTICE</summary>
I dedicate any and all copyright interest in this software to the
public domain. I make this dedication for the benefit of the public at
large and to the detriment of my heirs and successors. I intend this
dedication to be an overt act of relinquishment in perpetuity of all
present and future rights to this software under copyright law.
</details>

Fixes: #405 
I added a working bypass. clk.ink redirects automatically to clk.asia, so I bypassed clk.asia. Example link (leads to utorrent site): https://clk.ink/Utorrent

- [x] I made sure there are no unnecessary changes in the code*
- [x] Tested on Chromium- Browser OS
- [ ] Tested on Firefox

\* indicates required

### CODE REVIEW

**Comment by reashetyrr** [comment]
do you have an example link?

**Comment by unkn4wn** [comment]
> do you have an example link?

leads to french utorrent site:
https://clk.ink/Utorrent

File: src/js/injection_script.js [hunk 1]
```diff
@@ -2711,6 +2711,16 @@ domainBypass("acorta-link.com", () => {
     })
 })
 
+domainBypass("clk.asia", () => {
+    ensureDomLoaded(() => {
+      const token = document.querySelector('#link-view [name="token"]').value;
+      const decoded = atob(token.substring(token.indexOf("aH")));
```
Comment (reashetyrr) [inline src/js/injection_script.js]: dont we also want aHR0 here?

Comment (unkn4wn) [reply src/js/injection_script.js]: You're absolutely right. I updated and committed. That's the best "solution" till we find a better way to identify the beginning of a base64 string

**Review by reashetyrr — APPROVED** [review]
looks good to me, just merge it in! :)

**Pull request merged**